Modern layout

Ma1ithorne opened the pull request:
Hey, this version has the addition of the layout file BlizzModern.lua.
![Blizz Modern sample2](https://github.com/user-attachments/assets/d76387b4-e83a-4446-8b4c-3005ca5dd4c0)

Sammers21 commented:
@Ma1ithorne is the Pull request is ready to be reviewed? I'm trying to test the changes and i cant see the layout available for selection atm: 

![image](https://github.com/user-attachments/assets/c657ed2b-8d16-4ba5-a8d2-86ceb6d52092)

Ma1ithorne commented:
@Sammers21 Hey, you are correct - I forgot to add that to the .xml file here, fixed it now.

jomoroon commented:
wow this is a nice skin, any chance of adding as an option as part of your default addon sammers?

Sammers21 commented:
@Ma1ithorne i'm still having the same issue:
![image](https://github.com/user-attachments/assets/d056888c-cde6-42b3-8513-000323a7a76a)
The layout is not available for selection for me. Can you fix that? How do you making it work for yourself?

Ma1ithorne commented:
@Sammers21 That is strange, I re-downloaded this fork, moved the main addOn elsewhere temporarily, removed the '-master' at the end and it shows up for me. Do you have multiple versions in the addOn folder at once?
![image](https://github.com/user-attachments/assets/8d6c5acb-80df-42b2-9931-0ae93a00fbe3)

![image](https://github.com/user-attachments/assets/503e441e-1a95-4a6c-bcc1-c1e4b937be08)

Sammers21 commented:
@Ma1ithorne yes, i did exactly what you said. I will try again soon. Maybe i did something wrong.

Sammers21 commented:
@Ma1ithorne i have tried it again and its working fine for me.

Sammers21 commented on sArena.xml:
```diff
@@ -252,4 +253,4 @@
     </Frame>
-</Ui>
\ No newline at end of file
+</Ui>
```
Can you make it so this like is not changed?

Ma1ithorne replied:
> Can you make it so this line is not changed?

Hey, this should be fixed now.

Sammers21 commented on Layouts/BlizzModern.lua:
```diff
@@ -0,0 +1,263 @@
+local layoutName = "Blizz Modern"
+local layout = {}
+layout.name = "|cff00b4ffBlizz|r Modern by |cffFF7C0AMalithorn|r"
```
Can we change this to just `Blizz Modern`? I will mention your contribution in README and on the Curseforge description. Is that okay?

Sammers21 replied:
Or you want your name on the layout can just call it `Malithorn` without `Blizzard Modern`

Sammers21 commented:
@Ma1ithorne do the changes are we are good to merge/release

Sammers21 commented:
![image](https://github.com/user-attachments/assets/8366f456-5f16-4962-a25a-9e3dbad88200)
This is how the layout look in shuffle. Can you fix it?

Ma1ithorne commented:
> ![image](https://private-user-images.githubusercontent.com/16746106/382287806-8366f456-5f16-4962-a25a-9e3dbad88200.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2NDQ3NDksIm5iZiI6MTczMDY0NDQ0OSwicGF0aCI6Ii8xNjc0NjEwNi8zODIyODc4MDYtODM2NmY0NTYtNWYxNi00OTYyLWEyNWEtOWUzZGJhZDg4MjAwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTAzVDE0MzQwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZkODQ4MmM1MzQ3YmM5NzNkMTg1ZDU4ZDRiYmZjY2JlNmI2Yzg4ODkyZjNjZTM5OGMzNTgwY2YwZjQzNzVhMGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.IU_CMXI75deki6ARGAnXvM6cZtWn-zBey1sBqhHvr0o) This is how the layout look in shuffle. Can you fix it? @Ma1ithorne

Hey, fixed the default trinket/racial position.

Sammers21 reviewed (approved):
Looks good to me

Sammers21 reviewed (approved):
Looks good to me

Sammers21 commented:
@Ma1ithorne the changes has been released to curse forge. Thank you for the contribution and good job!
